### ISSUE TITLE
Has anyone tried importing a DLL in vs2019?

### ISSUE THREAD

**Issue by cornmonger** [electronicarts/CnC_Remastered_Collection#47]
Even when I match C++ / Linker options in VS 2019 with the original project (upgraded to v142), the mod errors out with Unable to load DLL.

**Comment by Crampers** [comment]
@alexlk42 made a repo you can clone directly should fix the issues

https://github.com/alexlk42/CnC_Remastered_Collection

**Comment by cornmonger** [comment]
> @alexlk42 made a repo you can clone directly should fix the issues
> 
> https://github.com/alexlk42/CnC_Remastered_Collection

I'm not having problems compiling and running the mod. I'm trying to import a DLL into the mod DLL.

**Comment by Capt-Nemo-** [comment]
I successfully converted TD dll to C++/CLI and I'm making calls into pure-C# dll that sits beside the main one, all of which should be about 1000% more hassle than what you are describing.
So it's most probably some configuration problem on your end rather than anything unusual the game does when it runs. I'm guessing you checked but are you sure your dll is compiled for x86?

**Comment by cornmonger** [comment]
Yeah, I'm guessing so as well. haha Yes, x86. It compiles and links fine. The DLL project is just a boiler plate vs2019 template with a function call and an export.

Originally I was attempting an embedded V8 library, but when that failed to work, I just tried importing a HelloWorld DLL.

**Comment by Crampers** [comment]
> 
> 
> > @alexlk42 made a repo you can clone directly should fix the issues
> > https://github.com/alexlk42/CnC_Remastered_Collection
> 
> I'm not having problems compiling and running the mod. I'm trying to import a DLL into the mod DLL.

Sorry! Completely misunderstood the issue.
